test/unit/support/azure/service_test.rb

```diff
@@ -0,0 +1,50 @@
+require_relative '../../test_helper'
+require_relative '../../../../libraries/support/azure/service'
+
+describe TestCredentials do
+  before do
+    @service = Object.new
+    @service.extend(Azure::Service)
+  end
+
+  it 'converts simple hashes to structs' do
+    struct = @service.structify({ name: 'John', age: 20 })
```

Comment:
Do we actually need the curly braces? (Doesn't Rubocop ding you for using them when they could be omitted?)